src/chain/chaininterface.rs

```diff
@@ -113,7 +155,7 @@ impl ChainWatchInterfaceUtil {
 	}
 
 	/// Checks if a given transaction matches the current filter
-	pub fn does_match_tx(&self, tx: &Transaction) -> bool {
+	fn does_match_tx(&self, tx: &Transaction) -> bool {
```

Comment:
Why skip this from being pub? If people use some compression-like block downloads (BIP 37, the neutrino stuff, some PIR scheme, whatever) then its probably still needed?

Reply:
yes, you are right. I make it pub again